llmfoundry/command_utils/train.py

```diff
@@ -584,7 +584,12 @@ def train(cfg: DictConfig) -> Trainer:
             )
 
         hf_checkpointer_callback = hf_checkpointer_callbacks[0]
-        hf_checkpointer_callback._save_checkpoint(trainer.state, trainer.logger)
+        hf_checkpointer_callback._save_checkpoint(
+            trainer.state,
+            trainer.logger,
+            upload_to_save_folder=True,
+            register_to_mlflow=False,
```

Comment:
i think it would be better for register to be true for this call

Reply:
yeah that's reasonable, can do that. fyi setting register=True would be a change in behavior from what was happening originally.

Reply:
yeah, when I added this I meant to check what would happen, and I intended for it to work with register in my head. and its not a breaking change, just does a bit more, so i think its ok.